src/cameras/search/hooks/use-search-input.js

```diff
@@ -0,0 +1,10 @@
+import { useContext } from 'cameras/context'
```

Comment:
Let's reorder our imports here.

Reply:
Done!

Reply:
![image](https://user-images.githubusercontent.com/23569938/129866720-8dc1c11e-feb3-455d-9950-258787502f81.png)
Umm... no, it's not.

Reply:
Done!